src/main/java/net/coderbot/iris/gl/texture/PixelFormat.java

```diff
@@ -37,6 +37,73 @@
 		}
 	}
 
+	public static Optional<PixelFormat> fromShaderpackString(String name) {
+		switch(name) {
```

Comment:
It feels like these are basically the same values that InternalTextureFormat has. Why not switch on that enum instead of switching on strings? Better yet, why not attach the expected PixelType to the InternalTextureFormat enum?

Reply:
I did this. We may want to add PixelType to InternalTextureFormat as well.